doc/tutorials/core/how_to_use_OpenCV_parallel_for_new/how_to_use_OpenCV_parallel_for_new.markdown

```diff
@@ -96,8 +96,8 @@ When looking at the sequential implementation, we can notice that each pixel dep
 
 @note Although values of a pixel in a particular stripe may depend on pixel values outside the stripe, these are only read only operations and hence will not cause undefined behaviour.
 
+C++11 standard allows a parallel implementation with a lambda expression:
```

Comment:
The following documentation describes `operator()` overloading and other steps of the old version. I propose not to remove pre-C++11 code, but just remove ifdefs for C++11.